test/integration/update-command/00-install.yaml

```diff
@@ -1,10 +1,12 @@
 apiVersion: kudo.dev/v1beta1
 kind: Instance
 metadata:
-  name: update-command-instance
   labels:
-    kudo.dev/operator: update-command-operator
+    controller-tools.k8s.io: "1.0"
```

Comment:
This test used a non-existing `update-command-operator` (I still wonder why it worked at all 🤔 ). It now uses the same `toy-operator` as everybody else

Reply:
No, it just creates an `Instance` named `update-command-instance` without caring to create an `Operator` or `OperatorVersion`. Seems like the instance controller can then update parameters without knowing about the instances O/OVs. Quite an edge-case, but probably what's actually being tested here. Changing the name of the instance shouldn't have any influence on the test.
But why remove the test step `02-add-new-param` here?

Reply:
That shouldn't be possible though, right? I would expect that creating an instance without O/OV would be forbidden...

Reply:
Yep, this shouldn't work @nfnt . We sort of casually reconciled from this situation previously, but without an `O/OV` there are no plans or param defaults so 🤷

Reply:
> Changing the name of the instance shouldn't have any influence on the test.
I didn't just change the name, but the references operator (L10)

> But why remove the test step 02-add-new-param here?
It basically did the same think as step 01 but with a different parameter name.

Reply:
Well, it worked before, but I agree that it shouldn't. Probably worked because it just applies resources and doesn't depend on the instance controller reacting to these changes. So it's an integration test for `kubectl kudo update` only, not the controller. Pretty dirty, IMO, but doing what it's supposed to do.

Reply:
Yeah, `02-add-new-param` is redundant in that context.